shell.nix

```diff
@@ -16,7 +16,12 @@ craneLib.devShell
       packages = with pkgs; with binPkgs; [
         cargo-dist
         cargo-flamegraph
+        # https://crane.dev/API.html?highlight=mold#cranelibcargonextest
+        # cargo-nextest
         rust-analyzer
+        # # Look into mold-wrapped or `pkgs.stdenvAdapters.useMoldLinker`
+        # # https://crane.dev/API.html?highlight=mold#cranelibdevshell
+        # mold-wrapped
```

Comment:
...however, is this bit also meant to be commented out? Is it intended as a "TODO" for a future PR?

Reply:
Yes, I'm happy to remove it for now, the TODO would be to use the mold linker elsewhere but whether to use it just for derivations or devshell included 🤷.